Jenkinsfile

```diff
@@ -1,14 +1,15 @@
 properties([parameters([
-  booleanParam(defaultValue: true, description: '', name: 'x86_64_linux'),
-  booleanParam(defaultValue: false, description: '', name: 'armv7_linux'),
-  booleanParam(defaultValue: false, description: '', name: 'armv8_linux'),
-  booleanParam(defaultValue: false, description: '', name: 'x86_64_macos'),
-  booleanParam(defaultValue: false, description: '', name: 'x86_64_win'),
+  booleanParam(defaultValue: true, description: 'Build iroha', name: 'iroha'),
+  booleanParam(defaultValue: false, description: 'Build iroha for ARMv7', name: 'armv7_linux'),
```

Comment:
Platform selectors should not be build type specific. E.g., checking `armv7_linux` should affect both `iroha` and `bindings` and build respective artifact. See previous Jenkins configuration